libraries/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Add standard modules
+file(GLOB standard_modules "${ROOT_DIR}/libraries/standard/*")
```

Comment:
How will this regex scale with multiple sub-directories in libraries? In the future, we will have an `aws` sub-folder for Jobs, Shadow, etc, libraries?
Can you update this regex to include _all_ CMakeLists.txt files within just the `libraries` folder?
Alternatively, this file can explicitly include the sub-directories containing the CMakeLists.txt as
```
add_subdirectory(standard/mqtt)
add_subdirectory(standard/http)
add_subdirectory(aws/jobs)
```

Reply:
Yes that is possible, but I would like to be explicit about the sub-folders we choose to include:
```cmake
file(GLOB standard_modules "${ROOT_DIR}/libraries/standard/*")
file(GLOB aws_modules "${ROOT_DIR}/libraries/aws/*")
foreach(module IN LISTS standard_modules aws_modules)
    if(IS_DIRECTORY "${module}" AND EXISTS "${module}/CMakeLists.txt")
        add_subdirectory(${module})
    endif()
endforeach()
```

Reply:
Okay that's fine. At this point, there exists no `aws` sub-folder, so need not add it in this PR :)